README.md

```diff
@@ -37,9 +37,12 @@ $ docker run --log-driver=sumologic --log-opt sumo-url=https://<deployment>.sumo
 ### Sumo Logic Options
 To specify additional logging driver options, you can use the `--log-opt NAME=VALUE` flag.
 
-| Option                      | Required? | Default Value | Description
-| --------------------------- | :-------: | :-----------: | -------------------------------------- |
+| Option                    | Required? | Default Value | Description
+| ------------------------- | :-------: | :-----------: | -------------------------------------- |
 | sumo-url                  | Yes       |               | HTTP Source URL
+| sumo-source-category      | No        |               | Source category to appear when searching on Sumo Logic by `_sourceCategory`. Use `{{Tag}}` as the placeholder of `tag` option. If not specified, the source category of HTTP source will be used
+| sumo-source-name          | No        |               | Source name to appear when searching on Sumo Logic by `_sourceName`. Use `{{Tag}}` as the placeholder of `tag` option. If not specified, it will be the container's name
+| sumo-source-host          | No        |               | Source host to appear when searching on Sumo Logic by `_sourceHost`. Use `{{Tag}}` as the placeholder of `tag` option. If not specified, it will be the machine host name
```

Comment:
"If not specified, it will be the machine host name" should be in "default value" column